code/modules/mob/mob_defines.dm

```diff
@@ -110,6 +110,7 @@
 	var/list/abilities = list()         // For species-derived or admin-given powers.
 	var/list/speak_emote = list("says") // Verbs used when speaking. Defaults to 'say' if speak_emote is null.
 	var/emote_type = 1		// Define emote default type, 1 for seen emotes, 2 for heard emotes
+	var/treadmill_speed = 1 //1 for most player things, simple animals get lower, xenos get higher
```

Comment:
>many lines of code.

Exactly 1 for every override of the proc.

>increases performance speed.

I'm willing to bet that due to the shitload of vars on mobs already finding the var could get slower, then again the same could go for procs.